src/App.js

```diff
@@ -1,13 +1,51 @@
 import React from 'react';
+import GoodsList from './components/GoodsList';
 
 import './App.scss';
 
-// import { getAll, get5First, getRed } from './api/goods';
-// or
-// import * as goodsAPI from './api/goods';
+import getResponse from './api/goods';
 
-const App = () => (
-  <h1>Dynamic list of Goods</h1>
-);
+class App extends React.Component {
+  state = {
+    goodsList: [],
+  }
+
+  getALlGoods = () => {
+    getResponse().then(goodsList => this.setState({ goodsList }));
+  }
+
+  getFirstFiveGoods = () => {
+    getResponse().then((goodsList) => {
+      const preparedList = goodsList
+        .sort((firstGood, secondGood) => (
+          firstGood.name.localeCompare(secondGood.name))).slice(0, 5);
+
+      this.setState({ goodsList: preparedList });
+    });
```

Comment:
It is better to have sorting and slicing operations in the api file.
Frontend components should receive data as prepared as possible.